### DISCUSSION TITLE
Error in visualization of vonmises stress

### DISCUSSION THREAD

**Discussion by echo1115** [idaholab/moose#19668] [Q&A Modules: Solid mechanics] [answered]
Hello everyone,

When analyzing circular plates with holes, i applied pressure to the internal holes. However, by using the postprocessing software ParaView, the visualization of vonmises stress is always close to zero, while the visualization of strain is reasonable.

![result_error](https://user-images.githubusercontent.com/69078335/146675775-6a55c039-892c-4df6-b9c3-8e20cebb0b88.JPG)
![result_2](https://user-images.githubusercontent.com/69078335/146675848-89bcb9a1-efbd-451f-8fd6-ef9175a2ac68.JPG)

**Comment by GiudGiud** [comment, marked as answer]
Hello

It seems to be exactly 0 here. What time are you plotting this at?

Guillaume
PS: please paste input files (in quotes for formatting) rather than post screenshots.

**Reply by GiudGiud** [reply]
@echo1115 any update on this?
